pages/introduction.vue

```diff
@@ -54,9 +51,65 @@
         transition.value = "slide-next";
         step.value++;
     };
+
+    setTimeout(function(){if(step.value == 0){nextStep();}},8000);
+
 </script>
 
 <style>
+
+.move-in1{
+    animation: movein 2s linear;
+}
+
+.move-in2{
+    animation: moveindelayed 4s linear;
+}
+
+.fade{
+    animation: fadein 5s linear;
+}
+
+@keyframes movein{
+    0%{
+        opacity: 0;
+    }
+    70%{
+        translate: 100vw;
```

Comment:
Da der Text an der linken Seite gebunden ist, wäre ein fade in von links glaube ich cooler. also -100vw an dieser stelle